Countries/United Kingdom/local.yaml

```diff
@@ -19,7 +19,7 @@ woo/woocommerce_price_num_decimals: '2' # Number of decimals
 # Tax rates
 woo/woocommerce_tax_classes: 'Reduced rate\nZero rate' # If more than one reduced rate add '\nReduced rate 2'
 wootax/1: # Standard rate
-    country: UK
+    country: GB
```

Comment:
@gitbot ping @dz0ny tomorrow

Reply:
@dmurko: you used `UK` everywhere, but the list that is linked in `template.yaml` says it needs to be `GB`. Pls check.

Reply:
![image](https://user-images.githubusercontent.com/239513/45432915-48b0a880-b6ab-11e8-9b57-4f580e5255e7.png)

Reply:
![image](https://user-images.githubusercontent.com/239513/45433121-c1b00000-b6ab-11e8-9890-9bba83aea73c.png)
![image](https://user-images.githubusercontent.com/239513/45433147-c96fa480-b6ab-11e8-809d-cfc1a27397f3.png)

Reply:
![image](https://user-images.githubusercontent.com/239513/45433225-f4f28f00-b6ab-11e8-9d62-99b12b0330e5.png)

It's GB :)